.github/contributing.md

```diff
@@ -1,51 +1,55 @@
 - Have you read the [code of conduct](code_of_conduct.md)?
 - Check out the [existing issues](https://github.com/interledger/rafiki/issues) & see if we [accept contributions](#types-of-contributions) for your type of issue.
 
-### Table of Contents <!-- omit in toc -->
+## Table of Contents <!-- omit in toc -->
 
 - [Types of contributions](#types-of-contributions)
   - [:mega: Discussions](#mega-discussions)
   - [:beetle: Issues](#beetle-issues)
   - [:hammer_and_wrench: Pull requests](#hammer_and_wrench-pull-requests)
 - [Working in the rafiki repository](#working-in-the-rafiki-repository)
-  - [Workspaces](#workspaces)
-    - [How to share scripts between workspaces?](#how-to-share-scripts-between-workspaces)
   - [Code quality](#code-quality)
     - [Linting](#linting)
     - [Formatting](#formatting)
     - [Testing](#testing)
     - [Commit hooks](#commit-hooks)
     - [Language](#language)
     - [CI](#ci)
+  - [Reporting Issues](#reporting-issues)
+  - [Submitting Pull Requests](#submitting-pull-requests)
+  - [Review Process](#review-process)
 
 ## Types of contributions
-You can contribute to Rafiki in several ways. 
+
+You can contribute to Rafiki in several ways.
 
 ### :mega: Discussions
+
 Discussions are where we have conversations about Rafiki.
 
 If you would like to discuss topics about the broader ecosystem, have a new idea, or want to show off your work - join us in [discussions](https://github.com/interledger/rafiki/discussions).
 
 ### :beetle: Issues
+
 We use GitHub issues to track tasks that contributors can help with. We haven't finalized labels yet for contributors to tackle. If you want to help with work related to an issue, please comment on the issue before starting work on it.
 
 If you've found something that needs fixing, search open issues to see if someone else has reported the same thing. If it's something new, open an issue. We'll use the issue to discuss the problem you want to fix.
 
 ### :hammer_and_wrench: Pull requests
+
 Feel free to fork and create a pull request on changes you think you can contribute.
 
 The team will review your pull request as soon as possible.
 
 ### :books: Documentation
-We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
 
+We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
```

Comment:
That's not where documentation lives now. We've moved that inside the rafiki repository: https://github.com/interledger/rafiki/tree/main/packages/documentation

Reply:
Done.